ssl/ssl_locl.h

```diff
@@ -2666,4 +2666,31 @@ void ssl_ctx_system_config(SSL_CTX *ctx);
 #  define ssl3_setup_buffers SSL_test_functions()->p_ssl3_setup_buffers
 
 # endif
+
+/*
+ * SSL_DEBUG support
+ */
+# if !defined(__STDC_VERSION__) || __STDC_VERSION__ < 199901L
+#  define SSL_DEBUG_data_dump ssl_debug_data_dump
+# else
+#  ifdef SSL_DEBUG
+#   define SSL_DEBUG_data_dump(...) ssl_debug_data_dump(__VA_ARGS__)
+#  else
+#   define SSL_DEBUG_data_dump(...)
+#  endif
+# endif
```

Comment:
Uhmmm, why do we need this?

Reply:
If `SSL_DEBUG` is not defined, the debug code is completely removed. Or, are you referring to something else?

Reply:
(If it's the `__STDC_VERSION__` check, that's based on code from test/testutil.h)

Reply:
Ah...  but you're not doing that for C90, so it still looks odd...

Reply:
This code is in test/testutil.h:
```
# if !defined(__STDC_VERSION__) || __STDC_VERSION__ < 199901L
#  define TEST_error         test_error_c90
#  define TEST_info          test_info_c90
# else
#  define TEST_error(...)    test_error(__FILE__, __LINE__, __VA_ARGS__)
#  define TEST_info(...)     test_info(__FILE__, __LINE__, __VA_ARGS__)
# endif
```
So, if THAT code is wrong... then... whoops!

Reply:
What happens with the definition of `SSL_DEBUG_data_dump` in C90 when `SSL_DEBUG` is undefined?